config.json

```diff
@@ -381,6 +381,16 @@
       "topics": [
 
       ]
+    },
+    {
+      "uuid": "1a8abaa2-04f5-0e80-7986-5fb33995971caeb1496",
+      "slug": "two-fer",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 3,
```

Comment:
I think 3 is reasonable. But can you put it somewhere near to the other ones of difficulty 3?

Because of #199 the placement doesn't need to be perfect, but at least it should be close...